packages/react/src/primitives/Message/useMessageContext.ts

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { MessageColorTheme } from '../types';
+
+export interface MessageContextProps {
```

Comment:
To align with existing context interfaces prefer `MessageContextType`

Reply:
Thanks, updated